### PR TITLE
Add "ansible-test sanity"

### PR DESCRIPTION
**Pull request by gundalow**
Based on https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml

### CODE REVIEW

**Comment by gundalow** [comment]
@JieX19 (and other maintainer) could you please check the settings to allow GitHub Actions (CI) run on this repo.

1) Please review the settings on https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection/settings/actions
2) If there is an "Approve/enable CI/GitHub Action" on this PR please click that.
3) The branch naming seems strange, is `master` isn't the default branch. Is development happening else where, and you are just copying the Python here?

Thanks.

**Comment by gotmax23** [comment]
> 3\. The branch naming seems strange, is `master` isn't the default branch. Is development happening else where, and you are just copying the Python here?

That seems to be the case. In fact, the version of the code here is different than the code on Ansible Galaxy.

File: .github/workflows/ansible-test.yml [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    name: Sanity (Ⓐ${{ matrix.ansible }})
+    strategy:
+      matrix:
+        ansible:
+          # It's important that Sanity is tested against all stable-X.Y branches
+          # Testing against `devel` may fail as new tests are added.
+          - stable-2.9
+          - stable-2.10
+          - stable-2.11
+          - stable-2.12
```
Comment (dmsimard) [inline .github/workflows/ansible-test.yml:37]:
```suggestion
          - stable-2.12
          - stable-2.13
```

**Comment by JieX19** [comment]
> 3. The branch naming seems strange, is `master` isn't the default branch. Is development happening else where, and you are just copying the Python here?

The 'master' branch is not the default branch. We've been transferring the administrative permissions of this account to our CM team, I will talk to them to set the 'master' branch as the default branch. We name the branch, for example,  fos_v6.0.0/galaxy_2.1.5, fos_v6.0.0/galaxy_2.1.4... in a sequential way for the users to find the versions they want.

The code on Github is just mirrored from our inside.

**Comment by gotmax23** [comment]
Why don't you have a single `main` branch and `git tag` each release? The current structure is unhelpful and irregular.

**Pull request merged**